python/pyspark/sql/functions.py

```diff
@@ -44,6 +44,20 @@ def _(col):
     return _
 
 
+def _create_zero_arg_function(name, doc=""):
```

Comment:
There was a lot of scoping issues. We couldn't get it to work with Josh when it wasn't in a function of it's own :/